src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({
+                'is-active': (selectedTab.id === tab.id),
+              })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                onClick={() => {
+                  if (tab.id !== selectedTab.id) {
+                    onTabSelected(tab);
+                  }
```

Comment:
it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here